src/Plugin/search_api/display/ListDisplayDeriver.php

```diff
@@ -26,8 +26,8 @@ public function getDerivativeDefinitions($base_plugin_definition) {
       foreach ($datasources as $datasource) {
         $entity_type = $datasource->getEntityTypeId();
         $bundles = $datasource->getBundles();
-        foreach ($bundles as $bundle_id => $label) {
-          $id = $entity_type . PluginBase::DERIVATIVE_SEPARATOR . $bundle_id;
+        foreach ($bundles as $id => $label) {
+          $id = $entity_type . PluginBase::DERIVATIVE_SEPARATOR . $id;
```

Comment:
What is this change? Leave `bundle_id`. Yo uare using the variable $id in the foreach and then overriding it inside the loop...